recommendations/tech_learning.md

```diff
@@ -1,40 +1,52 @@
+---
+title: How to get started on any of our tech stacks
+description: The getting started guides
+---
```

Comment:
This file looks drastic, but only because it now is ran through prettier because I added the yams front-matter to get it linked into the rest of the docs automatically /cc @erikdstock